src/network/networkctl.c

```diff
@@ -106,6 +106,7 @@ static void setup_state_to_color(const char *state, const char **on, const char
 
 typedef struct LinkInfo {
         char name[IFNAMSIZ+1];
+        char netdev_kind[64];
```

Comment:
64 is so arbitrary? maybe make this a "char*"? or alternatively define a `#define NETDEV_KIND_MAX 64`?